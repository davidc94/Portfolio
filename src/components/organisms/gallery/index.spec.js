import React from 'react';
import renderer from 'react-test-renderer';
import Gallery from './index';

describe('Gallery component', () => {
  it('render Gallery as expected', () => {
    const render = renderer.create(
      <Gallery />
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});