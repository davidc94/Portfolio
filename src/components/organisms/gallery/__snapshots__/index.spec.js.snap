// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery component render Gallery as expected 1`] = `
<div
  className="sc-bwzfXH hweCyD"
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
      "justifyContent": "flex-start",
    }
  }
  width="full"
>
  <div
    className="sc-bwzfXH hweCyD"
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "row",
        "justifyContent": "flex-start",
      }
    }
    width="full"
  >
    <div
      className="sc-bwzfXH cvQdJg"
      style={
        Object {
          "flex": 1,
        }
      }
    />
    <div
      className="sc-bwzfXH cvQdJg"
      style={
        Object {
          "flex": 20,
        }
      }
    >
      <p
        className="sc-bdVaJa iLKJWs"
        color="#2B303A"
        size="large"
      >
        Gallery
      </p>
    </div>
  </div>
  <div
    className="sc-bwzfXH hweCyD"
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "row",
        "justifyContent": "flex-start",
      }
    }
    width="full"
  >
    <div
      className="sc-bwzfXH cvQdJg"
      style={
        Object {
          "flex": 1,
        }
      }
    />
    <div
      className="sc-bwzfXH cvQdJg"
      style={
        Object {
          "flex": 200,
          "flexDirection": "column",
        }
      }
    >
      <p
        className="sc-bdVaJa kRyTSY"
      >
        Now that the boring stuff is out of the way, I also enjoy photography and shooting videos so here's a small collection of of videos and pictures from travelling ✈️.
      </p>
    </div>
  </div>
</div>
`;
